blocks/cards/cards.js

```diff
@@ -35,6 +36,9 @@ export default function decorate(block) {
         li.classList.add(`cards-card-bg-${div.textContent}`);
       } else if (div.children.length === 1 && div.querySelector('picture')) {
         div.className = 'cards-card-image';
+        if (linkName) {
+          div.querySelector('a').setAttribute('title', linkName);
```

Comment:
should we check `div.querySelector('a')` for null too or are we sure it is there?

Reply:
By the component logic it seems the anchor tag is there always but, just in case in the future the structure will change, I added the verification.